source/draw/gpu/opengl/backend/GLExtensions.ooc

```diff
@@ -0,0 +1,16 @@
+import egl/egl
+
+GLExtensions: class {
```

Comment:
Missing `version (!gpuOff)`? At least the non-gpu tests and builds fail for some reason.